paasta_tools/cli/cmds/status.py

```diff
@@ -158,7 +158,8 @@ def report_status_for_cluster(service, cluster, deploy_pipeline, actual_deployme
 
     if len(deployed_instances) > 0:
         status = execute_paasta_serviceinit_on_remote_master('status', cluster, service, ','.join(deployed_instances),
-                                                             system_paasta_config, stream=True, verbose=verbose)
+                                                             system_paasta_config, stream=True, verbose=verbose,
+                                                             ignore_ssh_output=True)
```

Comment:
I'm wary of what happens when SSH fails to connect. I'm assuming this change also hides that output (e.g "Connection refused..") and I'm not sure we have other messaging in place for these situations. Can you test this out to see what happens?